week_1/2_apr/practice/buttons.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
@codal-kcolaco please check all values of input tags and how we can use it with diffrent values to input tag.

Reply:
@abhi-codal  Yes